### PR TITLE
Mostly fixed broken links

### PR DESCRIPTION
**Pull request by sinkswim**
There were several broken links in README.md, MIDIPlayer.html and WhitneyMusicBox.html.
Also fixed the canvas.style.height property in WhitneyMusicBox.html.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,6 @@
 
 * [Basic](./examples/Basic.html) - the most basic implementation.
 * [MIDIPlayer](./examples/MIDIPlayer.html) - how to parse MIDI files, and interact with the data stream.
-* [MultipleInstruments](./examples/MultipleInstruments.html) - synth drum and piano playing together
```
Comment (sinkswim) [inline README.md]: MultipleInstruments example is not present anymore in the examples folder.

File: README.md [hunk 2]
```diff
@@ -16,7 +15,6 @@
 * [Spiral Keyboard](http://spiral.qet.me/) by Patrick Snels
 * [Ragamroll](http://online-compute.rhcloud.com/ragamroll/) by Mani Balasubramanian
 * [Gbloink!](http://gbloink.com/alpha/) by Phil Jones
-* [Watch & Repeat](http://watchandrepeat.com/) by Etay Luz
```
Comment (sinkswim) [inline README.md]: Watch & Repeat is not hosted anymore.

File: README.md [hunk 3]
```diff
@@ -42,7 +40,7 @@ MIDI.loadPlugin({
 });
 ```
 
-### [MIDI.Plugin.js](./js/midi/plugin.js) - Controls MIDI output
+### [MIDI.Plugin.js](./js/midi/plugin.webaudio.js) - Controls MIDI output
```
Comment (sinkswim) [inline README.md]: This should actually link to plugin.webmidi.js and plugin.audiotag.js as well.

File: README.md [hunk 4]
```diff
@@ -192,7 +190,7 @@ MIDI.setEffects([
 
 ## Libraries
 
-* [colorspace.js](./js/color/spaceW3.js): Color conversions, music isn&rsquo;t complete without!
+* [colorspace.js](./examples/inc/colorspace.js): Color conversions, music isn&rsquo;t complete without!
```
Comment (sinkswim) [inline README.md]: This is the only colorspace.js in the repo.

File: examples/MIDIPlayer.html [hunk 5]
```diff
@@ -55,7 +55,7 @@
...
 		<div id="title" style="background: rgba(255,255,0,0.5); position: relative;color: #000; z-index: -1;padding: 5px 11px 5px;">Loading API...</div>
 		<h3 style="margin: 0">What&rsquo;s all this hubub?</h3>
-		<p><b><a href="https://github.com/mudcube/MIDI.js">MIDI.js</a></b> ties together, and builds upon frameworks that bring MIDI to the browser. Combine it with <a href="https://github.com/gasman/jasmid">jasmid</a> to create a web-radio MIDI stream similar to this demo, or with <a href="https://github.com/mrdoob/three.js/">Three.js</a>, <a href="https://github.com/zz85/sparks.js/">Sparks.js</a>, or <a href="http://glsl.heroku.com/">GLSL</a> to create Audio/visual experiments. Piano/guitar
+		<p><b><a href="https://github.com/mudcube/MIDI.js">MIDI.js</a></b> ties together, and builds upon frameworks that bring MIDI to the browser. Combine it with <a href="https://github.com/gasman/jasmid">jasmid</a> to create a web-radio MIDI stream similar to this demo, or with <a href="https://github.com/mrdoob/three.js/">Three.js</a>, <a href="https://github.com/zz85/sparks.js/">Sparks.js</a>, or <a href="http://glslsandbox.com">GLSL</a> to create Audio/visual experiments. Piano/guitar
```
Comment (sinkswim) [inline examples/MIDIPlayer.html]: Updated GLSL domain.

File: examples/MIDIPlayer.html [hunk 6]
```diff
@@ -63,7 +63,6 @@ <h3 style="margin: 0">What&rsquo;s all this hubub?</h3>
 			<h3>Basic Examples</h3>
 			<ol>
 				<li><a href="./Basic.html" target="_blank">Basic (play note)</a></li>
-				<li><a href="./MultipleInstruments.html" target="_blank">Multiple Instruments</a></li>
```
Comment (sinkswim) [inline examples/MIDIPlayer.html]: MultipleInstruments example is not present in the examples folder.

File: examples/WhitneyMusicBox.html [hunk 7]
```diff
@@ -34,7 +34,7 @@
 <body>
 Whitney Music Box animation in HTML5.<br>
 Graphics code by <a href="https://github.com/jbum/Whitney-Music-Box-Examples">Jim Bumgardner</a>.<br>
-Audio created with <a href="https://github.com/mudx/MIDI.js">MIDI.js</a><br>
+Audio created with <a href="https://github.com/mudcube/MIDI.js">MIDI.js</a><br>
```
Comment (sinkswim) [inline examples/WhitneyMusicBox.html]: Was linking to old username/repo.

File: examples/WhitneyMusicBox.html [hunk 8]
```diff
@@ -102,7 +102,7 @@
 		onsuccess: function() {
 			var canvas = document.getElementById('mycanvas');
 			canvas.style.width = gw + "px";
-			canvas.style.width = gw + "px";
+			canvas.style.height = gh + "px";
```
Comment (sinkswim) [inline examples/WhitneyMusicBox.html]: App worked appropriately on most browsers thanks to "canvas.height = gh;" on line 107 anyways.

**Comment by mudcube** [comment]
Thanks for the fixes @sinkswim!

**Pull request merged**